source/analysis/AM2DNormalizationAB.cpp

```diff
@@ -30,7 +32,9 @@ AM2DNormalizationAB::AM2DNormalizationAB(const QString &outputName, QObject *par
 	canAnalyze_ = false;
 	dataName_ = "";
 	normalizationName_ = "";
-    cacheUpdateRequired_ = false;
+	cacheUpdateRequired_ = false;
```

Comment:
Some AB's seem to intitialize cacheUpdateRequired_ to true and some to false in the AB constructor. Is it implementation dependent, or should there be a standard way of initializing this value?